### DISCUSSION TITLE
Release v1.0.7

### DISCUSSION THREAD

**Discussion by BazzaCuda** [BazzaCuda/CustomMenu#20] [Announcements]
Windows 64-bit binaries

- added an "Open Config Window" menu item to the tray icon.

N.B. **CustomMenu** treats the entire area above the taskbar as the desktop and any right-click in that area activates the menu. Adding the new menu item encroaches on the desktop area above the taskbar. Consequently, I have reserved the last 30 pixels above the taskbar to accommodate the new menu item; otherwise, left-clicking the menu item would have been treated as a desktop left-click and **CustomMenu** would have ignored it.

As a result, there is now a 30-pixel strip across the entire screen, just above the taskbar, which won't activate **CustomMenu**.
However...the added bonus of this is that if you right-click anywhere along this 30-pixel strip, because **CustomMenu** is now ignoring it, the standard Windows desktop menu will be displayed. Try it out!

#SilverLinings :D

<hr /><em>This discussion was created from the release <a href='https://github.com/BazzaCuda/CustomMenu/releases/tag/v1.0.7'>Release v1.0.7</a>.</em>

**Comment by CoolHandDevil505** [comment]
Hi,
I have two issues :

1. I notice some mouse lag on the first opening of the menu after sys boot (sometimes after that too)
https://github.com/BazzaCuda/CustomMenu/assets/37642912/4c9009ec-21e6-4103-93b8-9639e22e3f2a

2. When I right-click on the systray icon, and left-click on Open Config item, nothing happens, and if I right-click it disables all mouse clicks on Windows globally…

Windows 11 22621.2215

**Comment by BazzaCuda** [comment]
Interesting.

1. The only time I've had mouse lag on a Windows 11 machine was because Microsoft Defender/Anti-virus was getting involved when it didn't need to. Do you have CustomMenu.exe listed in MD as an exception in its current folder?

2. Do you still have Windows changing the desktop wallpaper at intervals? I would recommend disabling that, at least temporarily. I can't replicate your issue, so we need to look at what's different between my machines and yours, and as we found out last time, Windows does some very strange things with its messaging in order to implement that facility.

3. A minor point from your video (thanks for that, btw) is that if you wish, you can delete the "Right-click me to open config" menu item. Right-clicking on any menu item opens Config.

**Reply by CoolHandDevil505** [reply]
1. Yes, I have an exception set for CustomMenu.
2. Yes, I still have. Sorry 😏 I think I'll try on my other machine and get back to you.
3. Yeah, I figured that out, but I like it here for now 😁
4. I tested the hotkey function, it is a major addition to the software I think. How is it handling Ctrl, Alt, Winkey and stuff? I managed to record Ctrl + < (226), but it only write 226 it hotkey file, and works only for the session. Once reloaded, only [<] activates the menu in this case, Ctrl is gone…

**Reply by BazzaCuda** [reply]
Thanks for the heads-up on the hotkey modifiers. This should now be fixed in the v1.0.8 release. You'll need to delete the contents of "hotkey" and relaunch to reselect your hotkey.
